package.json

```diff
@@ -28,6 +28,7 @@
     "@wagmi/core": "^2.6.16",
     "@web3modal/wagmi": "^4.1.6",
     "antd": "^5.9.0",
+    "bignumber.js": "^9.1.2",
```

Comment:
why do we need another bignumber package?
ethers and web3 both have them, they use bn.js under the hood

Reply:
Sure, can understand. 

Just be mindful that there may be incompatibility between bn.js and bignumber.js.

And we should never send a float to an EVM contract.